src/misc/fs_utils_posix.cpp

```diff
@@ -147,4 +151,182 @@ std::deque<std_fs::path> get_xdg_data_dirs() noexcept
 
 #endif
 
+// ***************************************************************************
+// Local drive file/directory attribute handling, WINE-compatible
+// ***************************************************************************
+
+constexpr int ro_permissions = S_IRUSR | S_IRGRP | S_IROTH;
```

Comment:
Constants are UpperCamelCase by convention. There are lots of new constant definitions here, so same comment for all of them. So `RoPermissions`, `XattrReadMask`, etc.

Reply:
OK, changed - I think all of them.